packages/scaffold-ui/src/partials/w3m-swap-input/index.ts

```diff
@@ -74,6 +74,7 @@ export class W3mSwapInput extends LitElement {
             placeholder="0"
             type="text"
             inputmode="decimal"
+            pattern="[0-9,.]*"
           />
```

Comment:
On swaps we are validating the input with a utility function where does some things like, when you type dot (.) first, it turns it to `0.`, can we use same utility on Send as well?